lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -144,8 +148,8 @@ newtype WalletPutData = WalletPutData
     } deriving (Eq, Generic, Show)
 
 data WalletPutPassphraseData = WalletPutPassphraseData
-    { oldPassphrase :: !(ApiT (Passphrase "encryption"))
-    , newPassphrase :: !(ApiT (Passphrase "encryption"))
+    { oldPassphrase :: !(ApiT (Passphrase "encryption-old"))
```

Comment:
Are these -old/-new types really needed?

Reply:
No, this has purely a self-documenting purpose. Like most of our phantom types actually.